### DISCUSSION TITLE
Add a config option to globally exclude columns

### DISCUSSION THREAD

**Discussion by nckrtl** [spatie/laravel-activitylog#1100] [Ideas]
In a project I'm working on we won't need the id, created_at and updated_at columns to be includes in the properties column upon a create action. This can be derived from the other columns of the same activity record. We now have now set the logExcept() for every model but we'll have to remember to set it every time we create a new models.

Would you be open for a PR that allows a config option `log_except` which contains an array of columns that will never be logged.

A full example:
```
'log_except' => [
    'id',
    'updated_at',
    'created_at',
],
```

**Comment by freekmurze** [comment]
Addressed in the v5 branch (#1452). There is now a `default_except_attributes` config option:

```php
// config/activitylog.php
'default_except_attributes' => ['id', 'created_at', 'updated_at'],
```

This is merged with model-specific `logExcept()` settings, so you no longer need to repeat common exclusions on every model. Feedback welcome.
